### ISSUE TITLE
Priority 1 accessibility issue: Search box tabs as links

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6561]
**What**

The ‘tab’ components such as ‘Library catalogue’ and ‘images’ have a link nested within a <button> element, which is not allowed and could cause confusion for users of assistive technology. Upon activation, the page refreshes and focus is taken to the beginning of the page. This is not the expected behaviour of tabs.

<img width="587" alt="Screenshot 2021-05-27 at 16 41 59" src="https://user-images.githubusercontent.com/7315126/119856225-777a1600-bf0a-11eb-9050-22f07c58d024.png">

Solution:
Ensure interactive components are not nested to avoid technical issues and confusion. If a link is the desired mechanism for the input, the <button> element should instead be replaced with a <div> and the links implemented as described in Issue ID: DAC_Tab_List_01. Otherwise, to ensure the operation of controls is predictable, the ARIA design pattern for tabs is recommended. This includes only one of the tabs in tab sequence. The next focusable component is the tab panel itself. Users can move focus between tabs using the arrow keys. The tablist should have an aria-label and contain the buttons marked-up with the role of ‘tab’. The tabs should have an ‘aria-controls’ attribute referencing the id of the corresponding tabpanel. The aria-selected attribute can be used to indicate the selected
tab. The role of ‘tabpanel’ should then be used with an ‘aria-labelledby’ referencing the corresponding tab. For more information, please visit WAI-ARIA Authoring Practices 1.1: Example of Tabs with Manual Activation.

**Where**

Search box
